views/publication/summary.templ

```diff
@@ -36,7 +36,7 @@ templ Summary(c *ctx.Ctx, p *models.Publication, opts SummaryOpts) {
 									}
 								</span>
 								if f := p.MainFile(); f != nil {
-									@AccessLevel(c, f)
+									@SummaryAccessLevel(c, f)
```

Comment:
I see that the candidaterecord publication preview here too starts too deviate too much from the publication summary in other places (which is more broadly used, but is older). I have rewritten that publication Summary for the older layout, and moved the newer publication preview into the candidaterecord view package.

See https://github.com/ugent-library/biblio-backoffice/pull/1484